scripts/_common.sh

```diff
@@ -4,6 +4,8 @@
 # COMMON VARIABLES
 #=================================================
 
+main_domain=$(cat /etc/yunohost/current_host)
```

Comment:
It could be useful to specify why we want the main domain here.

How about:

```suggestion
# We want Mattermost emails to be sent from the main domain
# (the one for which the emails headers are properly configured),
# and not the subdomain or secondary domain used for Mattermost.
main_domain=$(cat /etc/yunohost/current_host)
```